airbyte-local.sh

```diff
@@ -175,21 +180,38 @@ function writeDstCatalog() {
     cat "$tempdir/$src_catalog_filename" | jq ".streams[].stream.name |= \"${stream_prefix}\" + ." > "$tempdir/$dst_catalog_filename"
 }
 
+function loadState() {
+    if [ -s "$src_state_filepath" ]; then
+      cat "$src_state_filepath" > "$tempdir/$src_state_filename"
+    else
+      echo "State file did not exist or was empty."
```

Comment:
instead of this line, echo in the beginning: `echo "Using state file: $src_state_filename"`